.travis.yml

```diff
@@ -199,6 +199,10 @@ script:
     EOF
   - |
     cat > $ODBCSYSINI/.odbc.ini <<-EOF
+    [ODBC]
+    Trace     = yes
+    TraceFile = /tmp/odbc-driver-manager.log
```

Comment:
since we create those file on travis, can we can that they appeared after our tests?

Reply:
Yes, I can incorporate that in the upcoming travis changes:
https://github.com/ClickHouse/clickhouse-odbc/pull/198